packages/grafana-ui/src/components/Forms/FieldValidationMessage.tsx

```diff
@@ -22,6 +24,14 @@ export const getFieldValidationMessageStyles = stylesFactory((theme: GrafanaThem
       border-radius: ${theme.shape.borderRadius()};
       position: relative;
       display: inline-block;
+
+      a {
+        color: ${linkColor};
+      }
+
+      a:hover {
+        text-decoration: underline;
+      }
```

Comment:
as per conv on slack, think using contrastText + underline, and then remove underline on hover is a good solution here